README.md

```diff
@@ -92,11 +92,34 @@ export const config = {
 
 ### API Configuration
 
-If you wish to use the Electron APIs then you will need to import (or require) the preload and main scripts in your app. Somewhere near the top of your preload:
+If you wish to use the Electron APIs then you will need to import (or require) the preload and main scripts in your app. Somewhere near the top of your preload.
+To import node_modules in your `preload.js`, you have to disable the sandboxing in your `BrowserWindow` config.
+
+It is recommended to not disable sandbox mode in Production, so it can be set with an Environment Variable from your package.json.
+
+```json
+"wdio": "NODE_ENV=test wdio run wdio.conf.js "
+```
+
+Set the sandbox option depending on the NODE_ENV variable
 
 ```ts
-if (isTest) {
-  import('wdio-electron-service/preload');
+const isTest = process.env.NODE_ENV === 'test';
+
+new BrowserWindow({
+  webPreferences: {
+      sandbox: !isTest
+      preload: path.join(__dirname, 'preload.js'),
+  }
+  // ...
+});
+```
+
+Load `wdio-electron-service/preload` conditionally in your `preload.js`
+
+```ts
+if (process && process.env.NODE_ENV === 'test') {
```

Comment:
process refers to the current script, therefore I do not thing it can be falsy

Reply:
@erwinheitzman I think it is undefined in preload.js if `sandbox` is set to true